### DISCUSSION TITLE
Error event listener doesn't appear to be working

### DISCUSSION THREAD

**Discussion by rajputav** [jeffreylanters/react-unity-webgl#507] [Question and Answer]
### Please avoid duplicates

- [X] I checked [all open issues](https://github.com/jeffreylanters/react-unity-webgl/issues) and [discussions](https://github.com/jeffreylanters/react-unity-webgl/discussions) and none of them matched my problem.

### Language and Compiler

Vanilla JavaScript

### What environment are you using?

Server Side Renderering

### When does your problem occur?

When the Unity App is running

### What does your problem relate to?

The problem seems Module related

### React-Unity-WebGL Version

9.2.1

### React Version

17.0.2

### Unity Version

2022.2.9f1

### What happened?

Version 8.x.x. of the package supported error listeners: https://github.com/jeffreylanters/react-unity-webgl/blob/7b7aac91690b7cd3ed7968e82b20c9f732b04c43/documentation/versioned_docs/version-8.x.x/api/on-error.md#L4 . 

When trying to implement an error listener on Verizon 9.2.1 using the `addEventListener(‘error’)` API, it appears that no errors are caught detected. Is this intended behavior or is there an undocumented API that captures the proper usage of the error event listener? 

Thanks!

### Would you be interested in contributing a fix?

- [ ] yes, I would like to contribute a fix

**Comment by jeffreylanters** [comment]
Hi Avinash! Sadly Unity doesn't expose caught errors that well, they're instead using `window.error` to dump their errors to the console without any proper way to interfere. Previous versions of the module intercepted these error messages, but this was proven to be too hacky of a solution. -- So until Unity implements proper error handling, there isn't anything we can do really.

**Comment by rajputav** [comment]
Thanks for the prompt response! I see an [`initialisationError` prop](https://github.com/jeffreylanters/react-unity-webgl/blob/7b7aac91690b7cd3ed7968e82b20c9f732b04c43/module/source/hooks/use-unity-instance.ts#L62) in the newest package version... 

Could that be accessed via the following?  `const { unityProvider, initialisationError ...  } =
    useUnityContext(props.unityConfig)`
